EIPS/eip-5023.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: 5023
+title: Shareable non-transferable non-fungible token standard
+description: A standard interface for shareable non-transferable NFTs
+author: Jarno Marttila (@yaruno), Martin Moravek (@mmartinmo)
+discussions-to: https://ethereum-magicians.org/t/new-nft-concept-shareable-nfts/8681
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-28
+requires: 165
+---
+
+## Abstract
+
+This standard allows creation of tokens that can hold value, but cannot be exchanged for rival goods such as money, cryptocurrencies or other goods that hold monetary value. Shareability of tokens can be understood as re-minting of a token for a new recipient e.g. by making a copy of the token and by giving it away while retaining the original one. The sharing and its associated events allow construction of a graph describing who has shared what to which party.
+
+## Motivation
+
+Traditional NFT standards such as ERC-721 and ERC-1155 have been developed to introduce artificial digital scarcity and to capture rival value. NFT standards and implementations don't necessarily have to be exhibit scarcity and in certain use cases such as in ATARCA project we wish to avoid introduction of deliberate scarcity. 
```

Comment:
```suggestion
Traditional NFT standards such as ERC-721 and ERC-1155 have been developed to introduce artificial digital scarcity and to capture rival value. NFT standards and implementations don't necessarily have to exhibit scarcity and in certain use cases such as in ATARCA project we wish to avoid introduction of deliberate scarcity. 
```